rules/sinks/third_parties/sdk/pendo/javascript.yaml

```diff
@@ -11,3 +11,12 @@ sinks:
     patterns:
       - "rn-pendo-sdk|ngx-pendo"
     tags:
+
+  - id: ThirdParties.SDK.Pixel.Pendo
+    name: Pendo
+    domains:
+      - "pendo.io"
+    filterProperty: "code"
+    patterns:
+      - "((\\$)?(window|w|this)[.])?(pendo)([.]initialize)?[(].*"
```

Comment:
change this to 
`(.*[.])?(pendo)([.]initialize)?[(].*`

pendo is unique enough